lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -128,15 +130,12 @@ class UnusedCSSRules extends ByteEfficiencyAudit {
   }
 
   /**
-   * @param {!Object} stylesheetInfo The stylesheetInfo object.
+   * @param {StyleSheetInfo} stylesheetInfo The stylesheetInfo object.
    * @param {string} pageUrl The URL of the page, used to identify inline styles.
-   * @return {?{url: string, wastedBytes: number, totalBytes: number}}
+   * @return {LH.Audit.ByteEfficiencyResult}
    */
   static mapSheetToResult(stylesheetInfo, pageUrl) {
-    if (stylesheetInfo.isDuplicate) {
```

Comment:
it appears this is a holdover from old code. It doesn't appear to be set anywhere